src/omit.test.ts

```diff
@@ -12,13 +12,31 @@ describe('data first', () => {
     const result = omit(obj, ['a']);
     expect(result).toEqual({});
   });
+
+  test('non existing prop', () => {
+    // @ts-expect-error -- should not allow non existing props
+    const result = omit({ a: 1, b: 2, c: 3, d: 4 }, ['not', 'in'] as const);
+    expect(result).toEqual({ a: 1, b: 2, c: 3, d: 4 });
+  });
 });
 
 describe('data last', () => {
   test('omit', () => {
-    const result = pipe({ a: 1, b: 2, c: 3, d: 4 }, omit(['a', 'd'] as const));
+    const result = pipe(
+      { a: 1, b: 2, c: 3, d: 4 },
+      omit(['a', 'd', 'a'] as const)
```

Comment:
Why did you add the additional 'a' here? If this is to test something new can you create a separate test for it?

Reply:
I think it is just a typo